src/constant/walletProvider.ts

```diff
@@ -7,4 +7,6 @@ export enum ProviderAgent {
   // For legacy metamask
   Metamask = "metamask",
   Web3Auth = "web3auth",
+  Rainbow = 'rainbow',
+  Coinbase = 'coinbase wallet',
```

Comment:
I noticed Coinbase enum is declared in `SupportedEip6963Provider`, also `ProviderAgent.Coinbase` is not used anywhere.

Maybe remove?

Reply:
Yes, I think we can remove it. It may have been useful in the past, but not anymore.